src/udev/udev-builtin-path_id.c

```diff
@@ -543,19 +543,54 @@ static sd_device *handle_ap(sd_device *parent, char **path) {
         return skip_subsystem(parent, "ap");
 }
 
+static int find_real_nvme_parent(sd_device *dev, sd_device **ret) {
+        _cleanup_(sd_device_unrefp) sd_device *nvme = NULL;
+        const char *sysname, *end;
+        int r;
+
+        /* If the device belongs to "nvme-subsystem" (not to be confused with "nvme"), which happens
+         * when multipath is enabled, then the syspath is something like the following:
```

Comment:
there are two multipath around, no? nvme-specific multipath and general multipath, no?

maybe be explicit here, and always say "nvme multipath" to avoid and confusion